configure.ac

```diff
@@ -199,6 +199,17 @@ if test "x$enable_host_optimization" = "xno" ; then
 	ARCH_ABI=default-linux
 fi
 
+AC_ARG_ENABLE([wfxt-cpu-pause],
```

Comment:
ODP pull requests are not supposed to contain merge commits as ODP development in this repository is based on rebases and linear commit history. So could you please rebase your branch on top of the latest master instead of merging it.